lib/rbs/unit_test/with_aliases.rb

```diff
@@ -0,0 +1,143 @@
+# frozen_string_literal: true
+
+module RBS
+  module UnitTest
+    module WithAliases
+      include Convertibles
+
+      class WithEnum
+        include Enumerable
+
+        def initialize(enum) = @enum = enum
+
+        def each(&block) = @enum.each(&block)
+
+        def and_nil(&block)
+          self.and(nil, &_ = block)
```

Comment:
We should add `return WithEnum.new to_enum(__method__ || raise) unless block` here

Reply:
and accepts no block call. So it should be ok.